pixi_diff_to_markdown/cli.py

```diff
@@ -44,9 +45,13 @@ def main(
             help="Whether to hide tables in a collapsible element.", show_default=False
         ),
     ] = None,
-):
+) -> int:
     data = "".join(stdin.readlines())
-    data_parsed = Diff.model_validate_json(data)
+    try:
+        data_parsed = Diff.model_validate_json(data)
+    except ValidationError as e:
+        print(f"Invalid json passed to executable: \n{e}")
```

Comment:
Printing the exception gives a human-readable representation of the error. At least this would not show the traceback and not the exception itself.

```suggestion
        print(f"Invalid json passed to executable: \n{e}", file=sys.stderr)
```